internal/vcs/git/revisions.go

```diff
@@ -47,6 +49,13 @@ type ResolveRevisionOptions struct {
 	NoEnsureRevision bool // do not try to fetch from remote if revision doesn't exist locally
 }
 
+const labelEnsureRevision = "ensure_revision"
+
+var resolveRevisionCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+	Name: "src_resolve_revision_total",
+	Help: "The number of times we call internal/vcs/git/ResolveRevision",
+}, []string{labelEnsureRevision})
```

Comment:
Looks like `labelEnsureRevision` is used just once. Maybe we can directly use the string in this case?
```suggestion
}, []string{"ensure_revision"})
```